network/robus/src/robus.c

```diff
@@ -75,6 +76,15 @@ void Robus_Init(memory_stats_t *memory_stats)
     baudrate = DEFAULTBAUDRATE;
     // mask
     Robus_MaskInit();
+
+    // multicast mask init
+    ctx.ShiftMulticastMask = 0;
+    ctx.MulticastTargetMax = 0;
```

Comment:
Global variable name MulticastTargetNumer

Reply:
Agree, if you don't need those variables on other files you should put them as file global variables.